nemo/collections/common/tokenizers/sentencepiece_tokenizer.py

```diff
@@ -221,8 +213,9 @@ def create_spt_model(
 
     cmd = (
         f"--input={data_file} --model_prefix={output_dir}/tokenizer "
-        f"--vocab_size={vocab_size - len(vocab)} "
+        f"--vocab_size={vocab_size} "
         f"--shuffle_input_sentence=true --hard_vocab_limit=false "
```

Comment:
I didn't work with this SPE code, not sure if it's ok to delete these lines. 
Maybe leave for a separate PR and discuss with @AlexGrinch before this change

Reply:
i removed the special tokens from create_spc_tokenizer, it works fine and it was never used for that anyway